services/datalinker/templates/ingress.yaml

```diff
@@ -0,0 +1,35 @@
+{{- if .Values.ingress.enabled -}}
+{{- $fullName := include "datalinker.fullname" . -}}
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  name: {{ $fullName }}
+  labels:
+    {{- include "datalinker.labels" . | nindent 4 }}
+  {{- with .Values.ingress.annotations }}
```

Comment:
Nit: Move this line down to where the annotations are actually used to make it match the pattern of other services.

Reply:
done